### PR TITLE
FW Proposal PR (not main)

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

Proposed change of union

### CODE REVIEW

**Review by MariusStorhaug — APPROVED** [review]
Guess @jontreynes should approve too?

**Comment by jontreynes** [comment]
> Guess @jontreynes should approve too?

yea, I just pushed a change to this branch to fix the linter natural language errors

**Comment by jontreynes** [comment]
ah wait looks like it was already merged - no worries I'll fix on my branch before merging to main

**Pull request merged**
